src/libutil/args.cc

```diff
@@ -29,7 +29,16 @@ void Args::removeFlag(const std::string & longName)
 
 void Completions::add(std::string completion, std::string description)
 {
-    assert(description.find('\n') == std::string::npos);
+    // strip whitespace/empty lines from the front of the description
+    description.erase(0, description.find_first_not_of(" \t\n"));
```

Comment:
This can be done slightly easier with `trim()` in libutil.

Reply:
Fixed it. I've only started learning Nix on monday. So I'm unfamiliar with the codebase and style. Thank you for the fast response and awesome work on Nix!